build.gradle

```diff
@@ -672,7 +677,7 @@ jlink {
     // but it was removed in the newer releases.
     // The pom.xml associated with such a non-modular artifact does not mention that the artifact depends on the removed code
     // (because the artifact was published when this code was still available in the JDK).
-    forceMerge "controlsfx", "bcprov", "jaxb", "istack", "stax", "log4j"
```

Comment:
We have to try if logging works for a library logging using log4j. I don't know how to find out which one is it 😅🙈

Reply:
I played around in Jabref and tested different dialogs and actions and so far I did not stumble across any error